### ISSUE TITLE
Impact of including climate models with high ECS in the ensemble

### ISSUE THREAD

**Issue by lpcaron** [ECCC-CCCS/CMIP6-BCCAQ-Quality-Control#37]
After reading a recent paper by [Hausfather et al. (2022)](https://www.nature.com/articles/d41586-022-01192-2), we decided to apply their methodology to select the CMIP6 models on the CCDP ensemble that we had. In blue is all 26 models we currently have for CCDP and in red is the 15 models which have a transient climate response (TCR) that falls in the ‘’likely’’ category as defined in the latest IPCC report. Each plot is a different location in Canada. We can see some substantial differences, in particular for the 90th percentile (as expected).  This result is also consistent with the result reported in issue #19.

This suggests that we have to think carefully about how we create our CMIP6 ensemble.

![Hot_models](https://user-images.githubusercontent.com/37218446/170129121-104c120b-f9e7-4f2e-a43d-fbb787834870.png)

@JeremyFyke @juliettelavoie @tlogan2000 @cpomer10 @ssobie @elaine-cccs

**Comment by cpomer10** [comment]
@btam001 this may be of interest to you too

**Comment by JeremyFyke** [comment]
This is great @lpcaron.  Do you/Ouranos potentially have plans to carry out same analysis on some more extreme indices, e.g. ClimDex or similar?  I suspect it would be pretty interesting to see these...

**Comment by btam001** [comment]
This is great @lpcaron! It's very interesting to see the results

**Comment by juliettelavoie** [comment]
@JeremyFyke  I made a few more indices (this is 30 year mean, above was showing every year):

![image](https://user-images.githubusercontent.com/29236239/170362784-e9e21d48-a584-4891-bfde-f25fa0795fa1.png)

**Comment by lpcaron** [comment]
Would it be worth discussing at the PWG (or resuscitate the CMIP6 group) whether we want to go ahead with the original ensemble or if we want to modify that ensemble somehow before publishing the new data online?

**Comment by cpomer10** [comment]
Yes we can add this topic and the possibility of using a different ensemble to the next PWG agenda and invite others who may be interested. In the meantime I think we will continue to move forward with original ensemble but will make sure that it is clear that some of the models are "hot" and provide relevant guidance (incl. that we may be adding a reduced ensemble). These figures would certainly be helpful there.

**Comment by juliettelavoie** [comment]
Even though, we will not be putting the Hausfather ensemble on climatedata.ca for now, we calculated the new ensemble stats on the 30y mean of the 16 selected models for all indices. Here are a few figures of the difference between the complete ensemble and the the hausfather ensemble. We are showing the 10th, 50th and 90th percentiles. Reach out if you want us to send you the full dataset.

![image](https://user-images.githubusercontent.com/29236239/180282317-7a155ee2-d968-4a0f-8413-4595538f2ac0.png)
![image](https://user-images.githubusercontent.com/29236239/180282334-1da4038b-74a9-40d9-98e6-1ed8edc695bf.png)
![image](https://user-images.githubusercontent.com/29236239/180285757-cd9fb3f3-d718-4680-ae4d-c6ed5fac0c4a.png)

![image](https://user-images.githubusercontent.com/29236239/180285181-7f893d0b-052c-44bc-a5fd-befdf3bc172f.png)
![image](https://user-images.githubusercontent.com/29236239/180285188-3bfdeffa-cb65-4260-b459-b2c9c245bb98.png)

**Comment by tlogan2000** [comment]
Merci Juliette!
